wfc3tools/embedsub.py

```diff
@@ -101,18 +101,23 @@ def embedsub(files):
         # regular size image
         flt[0].header['SUBARRAY'] = False
 
+        # Make a duplicate of the input file and name it variable full
+        os.system(f"cp {filename} {full}")
```

Comment:
i'm not sure you have to do this - i think if you open the input file in read-only mode, modify its hdu list, you can use the hdu.write_to method to write that out to a new output file